tracer/test/Datadog.Trace.Security.Unit.Tests/Datadog.Trace.Security.Unit.Tests.csproj

```diff
@@ -1,5 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <ItemGroup>
+    <None Remove="IAST\Tainted\evidence-redaction-suite.yml" />
+  </ItemGroup>
+
```

Comment:
tiny nit: move this next to the`<EmbeddedResource>`
```suggestion
```

Reply:
Fixed in 7580be933ab5b431311d4b3f109543549fa1659a